dip-template/runtimes/dip-consumer/src/lib.rs

```diff
@@ -260,6 +234,7 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type SelfParaId = ParachainInfo;
 	type XcmpMessageHandler = ();
+	type ConsensusHook = cumulus_pallet_parachain_system::ExpectParentIncluded;
```

Comment:
This needs to be changed to Fixed Velocity

Reply:
fixed in: [ca13129](https://github.com/KILTprotocol/kilt-node/pull/627/commits/ca13129273c0abcfb2dbeea57c5efeaab730693a)